content/blog/the-new-github-auth-flow.md

```diff
@@ -0,0 +1,61 @@
+---
+title: The New GitHub Auth Flow
```

Comment:
```suggestion
title: Upgrade Notice: TinaCMS + GitHub Packages
```
Maybe this instead :sweat_smile:

Reply:
Filename too? Maybe `upgrade-notice-github-auth.md`